_locales/en/messages.json

```diff
@@ -517,7 +517,7 @@
     "message": "Download anonymized logs"
   },
   "DescriptionDownloadlogs": {
-    "message": "Floccus logs all its actions in a log file that you can examine yourself or send to the developers for debugging purposes. When sending anonymized logs, make sure to still check the downloaded file for sensitive data that might not have been caught by the anonymization process."
+    "message": "Floccus logs all its actions in a log file that you can examine yourself or send to the developers for debugging purposes. This log file is also available in an anonymized version that hides the titles and links of your bookmarks. When sending anonymized logs, make sure to still check the downloaded file for sensitive data that might not have been caught by the anonymization process."
```

Comment:
Please do not add multiple commits for different features/fixes to the same branch / Pull request as it makes reviewing stuff harder. Ideally, each pull request goes into a separate branch. ;)

Reply:
I'm sorry, I didn't know this "branch" thing.. 
How can I create a new branch when I have a new feature that i want to ask pull request ?

Reply:
Don't worry, I'll stop adding commits while I didn't sort this branch problem out.
I'll investigate on the web.